### PR TITLE
feat: Adding the home page and tests for it

### PR DESCRIPTION
**Pull request by sergiorodriguezgarcia**
New home funtionality and tests for it.

### CODE REVIEW

File: webapp/src/pages/Root.jsx [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
 export default function Root() {
-    return <p>Proof of concept</p>
+    const navigate = useNavigate();
+    const { t } = useTranslation();
+
+    return (
+        <Center display={"flex"} flexDirection={"column"} w={"100wh"} h={"100vh"}
+            bg={"blue.50"} justifyContent={"center"} alignItems={"center"}>
+            <Heading as="h1" color="blue.400">{"WIQ-EN2B"}</Heading>
+            <p>Welcome to the WIQ-EN2B page</p>
```
Comment (jjgancfer) [inline webapp/src/pages/Root.jsx:16]: Please, localize this part.

**Review by jjgancfer — CHANGES_REQUESTED** [review]
Looks good! We will probably have to revisit it later to redirect in case the user is already logged, but for now will do. Nevertheless, please localize the part marked.

**Comment by sergiorodriguezgarcia** [comment]
I think is a good approach, I will change and upload it.

**Review by jjgancfer — APPROVED** [review]
Since the requested change was made, I'll approve the pull request.

**Review by gony02 — APPROVED** [review]
Everything seems fine, good job!!!

**Pull request merged**